data_dictionary.md

```diff
@@ -23,14 +28,34 @@
 }
 ```
 
-The Activity Stream page will ping (HTTPS POST) [Onyx](https://github.com/mozilla/onyx) every time the page loses focus.
+# Example Activity Stream `event` Log
+
+```json
+{
+  "action": "activity_stream_event",
+  "addon_version": "1.0.0",
+  "click_position": 4,
+  "client_id": "374dc4d8-0cb2-4ac5-a3cf-c5a9bc3c602e",
+  "date": "2016-03-07",
+  "ip": "10.192.171.13",
+  "locale": "en-US",
+  "source": "top_sites",
+  "event": "delete",
+  "tab_id": 3,
```

Comment:
Each `tab_id` is a string, e.g. "1-3", which stands for a combination of window id and tab id.